Lib/email/utils.py

```diff
@@ -215,6 +215,14 @@ def parsedate_to_datetime(data):
 
 
 def parseaddr(addr):
+    """
+    Parse address into its constituent 
+    realname and email address parts
+    
+    Returns a tuple of realname and email address,
+    unless the parse fails,
+    in which case a 2-tuple of ('', '') is returned.
```

Comment:
Wrap this into a paragraph with a max width of 79 characters.  Change 'Returns' to 'Return' (imperative voice), and likewise "in which case" becomes "in which case return" and drop 'is returned'.